.github/workflows/ruby.yml

```diff
@@ -1,19 +1,42 @@
+---
 name: Ruby
 
-on: [push, pull_request]
+on:
```

Comment:
that's the default config we use in other vox pupuli gems. run on all PRs and on merges to the main branch